recipes/libverto/all/test_package/CMakeLists.txt

```
@@ -1,8 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package C)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+find_package(libverto REQUIRED core CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.c)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_link_libraries(${PROJECT_NAME} PRIVATE libverto::verto)
```

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE libverto::liberto)
```

link everything so that we can see whether it works.